packages/bappo-components/src/components/Badge/Badge.tsx

```diff
@@ -1,26 +1,21 @@
-// @flow
-
 import * as React from 'react';
 
 import { styled } from '../../apis/Style';
 import Text from '../../primitives/Text';
 import View from '../../primitives/View';
 
 type Props = {
-  number?: number,
+  number?: number;
+  testID?: string;
 };
 
-const Badge = ({ number }: Props) => {
+export default function Badge({ number, testID }: Props = {}) {
```

Comment:
Hi Emil, I am sorry I forget to mention  in Badge, there is also a empy object

Reply:
Thanks Jon, I've removed the empty default value from Badge too.